src/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <ul>
+      {props.contactDetails.map((contact, index) => {
+        return <li key={index}>{contact}</li>;
```

Comment:
The styling could potentially be nicer on your footer, for example the list items don't really need bullet points. This is me being picky though :)